views/default/forms/admin/site/settings/content.php

```diff
@@ -60,4 +60,13 @@
 	'value' => 1,
 ]);
 
+$body .= elgg_view_field([
+	'#type' => 'number',
+	'#label' => elgg_echo('config:content:comments_per_page'),
+	'name' => 'comments_per_page',
+	'value' => elgg_get_config('comments_per_page'),
+	'min' => 1,
+	'step' => 1,
```

Comment:
why set this, as 1 is (probably) the default

Reply:
kept inline with default list length setting